packages/jest-preset-default/jest-preset.js

```diff
@@ -26,5 +27,5 @@ module.exports = {
 	transform: {
 		'^.+\\.[jt]sx?$': require.resolve( 'babel-jest' ),
 	},
-	verbose: true,
+	verbose: false,
```

Comment:
`false` is the default value so it can be safely removed. It only slows down test execution and it should never be enabled in the default preset.

Reply:
Removed with https://github.com/WordPress/gutenberg/pull/33287/commits/cbf38a471eabe18f74fd7762b2088a4467fe9b2a